components/Form/index.js

```diff
@@ -0,0 +1,332 @@
+import styled from "styled-components";
+import { css } from "styled-components";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+export default function Form({ onSubmit }) {
+  const [sleep, setSleep] = useState(true);
+  const [hygiene, setHygiene] = useState(false);
+  const [medicine, setMedicine] = useState(false);
+
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    for (const property in data) {
+      if (data[property].trim() === "") {
+        return alert("Bitte gib einen Text in das Feld ein!");
+      }
+    }
+
+    onSubmit(
```

Comment:
I think, it would be better to leave the name of your function as it was (`addFacility`). This is more descriptive and easier to understand than calling a function called `onSubmit` in the `handleSubmit` function :)